src/App.tsx

```diff
@@ -1,25 +1,44 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.css';
-import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const maxItems = 42;
 
 export const App: React.FC = () => {
+  const [value, setValue] = useState<string>('5');
+  const [column, setColumn] = useState<number>(0);
+
+  const cheakChenge = (event: ChangeEvent<HTMLSelectElement>) => {
+    setValue(event.target.value);
+    setColumn(0);
+  };
+
+  const Start = column * Number(value);
+  const End = Math.min(Start + Number(value), maxItems);
+  const Items: string[] = [];
+
+  for (let i = Start + 1; i <= End; i++) {
+    Items.push(`Item ${i}`);
+  }
```

Comment:
same here, just return the getNumbers logic and slice the array from start to end